exercises/practice/allergies/tests/allergies.rs

```diff
@@ -335,7 +335,6 @@ fn allergic_to_everything_cats() {
 
 #[test]
 #[ignore]
-
```

Comment:
The previous version of the template for `allergies` seems to have generated unnecessary whitespace. This change fixes that (by accident).

Reply:
Funnily enough, I just noticed that yesterday!